mimesis/data/de/food.json

```diff
@@ -147,15 +147,26 @@
     "Stippgrütze"
   ],
   "drinks": [
```

Comment:
Please, format file using http://jsoneditoronline.org

Reply:
Sorry for the inconvenience, didn't check for unwanted whitespaces and tabs which seem to got created by intellij.
formated in commit 9849aef

Reply:
@schwedenmut Great! Danke!